syntax = "proto3";

service FaceRecognition {
  rpc GetFacePosition(ImagePair) returns (Rectangle) {}
}

message ImagePair {
    string personFace = 1;
    string crowdImage = 2;
}

message Rectangle {
    int32 code = 1;
    string message = 2;
    bytes listPosition = 3;
}